cli/file_watcher.rs

```diff
@@ -76,6 +76,19 @@ where
   }
 }
 
+async fn error_handler2<F>(watch_future: F) -> i32
```

Comment:
Maybe a slightly more descriptive name would be beneficial such as `result_handler` ?